.stickler.yml

```diff
@@ -0,0 +1,3 @@
+
+linters:
+  stylelint:
```

Comment:
- The original stickler file is like this:
https://github.com/microverseinc/linters-config/blob/d3f1673deb9f566bb93216631015d0eb69c64c6e/css/.stickler.yml#L1-L3.
:pray:  use it.